homeassistant/components/intellifire/config_flow.py

```diff
@@ -0,0 +1,87 @@
+"""Config flow for Intellifire integration."""
+from __future__ import annotations
+
+import logging
+from typing import Any
+
+from intellifire4py import IntellifireAsync
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.exceptions import HomeAssistantError
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required("host"): str,
+        vol.Required(
```

Comment:
Dont ask for the name.
 Keep the flow to minimum and then users can rename in the UI